### PR TITLE
Issue 12 5

### PR DESCRIPTION
**Pull request by carlosTorres07**

### CODE REVIEW

File: dal/src/main/java/com/craftsmanship/tfm/dal/config/DalConfig.java [hunk 1]
```diff
@@ -1,26 +1,43 @@
...
-        GrpcServer grpcServer = new GrpcServer(service);
+
+        ItemPersistenceService itemService = new ItemPersistenceService(itemDAO, entityConversion);
+        OrderPersistenceService orderService = new OrderPersistenceService(orderDAO, entityConversion);
+
+        List<BindableService> services = new ArrayList<BindableService>();
+        services.add(itemService);
+        services.add(orderService);
+
+        GrpcServer grpcServer = new GrpcServer(itemService);
```
Comment (kikogolk) [inline dal/src/main/java/com/craftsmanship/tfm/dal/config/DalConfig.java]: You are adding just the itemService, aren't you?

**Review by kikogolk — COMMENTED** [review]
Please, check comment

File: dal/src/main/java/com/craftsmanship/tfm/dal/config/DalConfig.java (hunk 1 above)
Comment (carlosTorres07) [reply dal/src/main/java/com/craftsmanship/tfm/dal/config/DalConfig.java]: Yes, you are right, it is a bug..

Comment (kikogolk) [reply dal/src/main/java/com/craftsmanship/tfm/dal/config/DalConfig.java]: great

**Review by kikogolk — APPROVED** [review]

**Pull request merged**